dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OrganizationIT.java

```diff
@@ -2823,6 +2823,11 @@ void testAddCategoriesAndAddRemoveEntry() {
         addAdminToOrg(ADMIN_USERNAME, "dockstore");
 
         final io.dockstore.openapi.client.ApiClient webClientUser = getOpenAPIWebClient(OTHER_USERNAME, testingPostgres);
+        EventsApi eventsApi = new EventsApi(webClientUser);
```

Comment:
With all the qualified package imports, may be better to just move this out to `OpenAPIOrganizationIT`, not a blocker for 1.14 but create a ticket with some clean-up actions including this one if this PR is merged as-is

Reply:
Can link to https://ucsc-cgl.atlassian.net/browse/SEAB-4697

Reply:
Created: https://ucsc-cgl.atlassian.net/browse/SEAB-5425